pkg/audit/verifier.go

```diff
@@ -57,26 +57,18 @@ func NewVerifier(transport transport.Client, overlay overlay.Client, id provider
 	return &Verifier{downloader: newDefaultDownloader(transport, overlay, id)}
 }
 
-func (d *defaultDownloader) dial(ctx context.Context, storageNode *pb.Node) (ps client.PSClient, err error) {
-	defer mon.Task()(&ctx)(&err)
-	c, err := d.transport.DialNode(ctx, storageNode)
-	if err != nil {
-		return nil, err
-	}
-	return client.NewPSClient(c, node.IDFromString(storageNode.GetId()), 0, d.identity.Key)
-}
-
 // getShare use piece store clients to download shares from a given node
 func (d *defaultDownloader) getShare(ctx context.Context, stripeIndex, shareSize, pieceNumber int,
-	id client.PieceID, pieceSize int64, node *pb.Node, authorization *pb.SignedMessage) (s share, err error) {
+	id psclient.PieceID, pieceSize int64, _node *pb.Node, authorization *pb.SignedMessage) (s share, err error) {
```

Comment:
Aliasing of package names is frowned upon because it makes it more difficult to understand things when moving from package to package. Consistency is key.

Reply:
OK. I see. What's your opinion about `fromNode` vs `_node`? Are you fine using `_node` as param/variable names?

Reply:
Just `from` would do, but `fromNode` is fine as well.